metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1249,6 +1250,24 @@ class MetalsLanguageServer(
             val msg = s"Invalid arguments: $args. Expecting: $argExample"
             Future.failed(new IllegalArgumentException(msg)).asJavaObject
         }
+      case ServerCommands.NewScalaWorksheet() =>
+        val args = params.getArguments.asScala
+        def f(dir: URI, name: String) = Future {
+          val path = dir.toString.toAbsolutePath.resolve(name + ".worksheet.sc")
+          //TODO: report failure, like file already exists?
```

Comment:
Maybe we can just use `client.showMessage`?

Reply:
Done + `scribe.error` added

Reply:
![Screenshot from 2020-01-24 17-10-31](https://user-images.githubusercontent.com/10850363/73084220-dcfc6380-3ecc-11ea-8437-bd2b2d020c90.png)